indico/modules/events/ical.py

```diff
@@ -87,7 +89,8 @@ def generate_event_component(event, user=None):
         as_list=True
     ):
         data.update(update)
-    component.add('description', data['description'])
+    if data['description']:
+        component['description'] = data['description']
```

Comment:
does this work even if there is no description yet? (can this happen?)

Reply:
It works, but it should not happen except for entities without any URL, as the description will include that as a minimum now.